get-started/fine-tuning/api.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Fine-tuning with the API
+description: Learn how to programmatically create custom models tailored to your specific tasks by fine-tuning foundation models with your own data using the kluster.ai API.
+---
+
+# Fine-tuning with the API
+
+The kluster.ai API lets you automate and integrate fine-tuning into your development workflows. You can create, manage, and monitor fine-tuning jobs directly from your code, making it easy to customize models for your specific needs.
+
+This guide provides a practical overview of the fine-tuning process using the API. It covers the required data format, how to upload your dataset, and how to launch and monitor a fine-tuning job. For a step-by-step walkthrough, see the linked tutorial in the tips below.
+
+## Prerequisites
+
+Before getting started with fine-tuning, ensure you have the following:
+
+- **kluster.ai account**: Sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one.
+- **kluster.ai API key**: After signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide.
+- **Prepared dataset**: You need data formatted according to kluster.ai's requirements for fine-tuning (detailed below).
+
+## Supported models
+
+kluster.ai currently supports fine-tuning for two base models:
+
+- [klusterai/Meta-Llama-3.1-8B-Instruct-Turbo](https://docs.kluster.ai/get-started/models/){target=_blank}
+- [klusterai/Meta-Llama-3.3-70B-Instruct-Turbo](https://docs.kluster.ai/get-started/models/){target=_blank} 
+
+!!! info
+    You can query the [models endpoint](https://docs.kluster.ai/api-reference/reference/#list-supported-models){target=_blank} in the API and filter for the tag "fine-tunable."
+
+## Fine-tuning workflow
+
+The process of fine-tuning a model using the kluster.ai API involves several key steps:
```

Comment:
I would add the steps first, like we do with Batch, so that the readers understand them before doing a deep dive into each. Similar to -> https://docs.kluster.ai/get-started/start-building/batch/#batch-job-workflow-overview